src/api-service/__app__/onefuzzlib/pools.py

```diff
@@ -195,6 +214,12 @@ def can_process_new_work(self) -> bool:
             self.stop()
             return False
 
+        if self.state in NodeState.ready_for_reset():
+            logging.info(
+                "can_schedule should be recycled.  machine_id:%s", self.machine_id
```

Comment:
What does "recycled" mean here?

Do we really want to have this message be identical to the one below?

Do we need to call `stop()` here, too?

Reply:
1. updated the messages to say reset or deletion as appropriate.
2. no, we don't want it to say stop here, as ready_for_reset() indicates the nodes have already seen the stop message.